agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml

```diff
@@ -14,6 +14,8 @@ rules:
   - limitranges
   - namespaces
   - nodes
+  - nodes/metrics
+  - nodes/proxy
```

Comment:
@fremmi this will turn useful for https://sysdig.atlassian.net/browse/SMPROD-5959